core/src/main/scala/spinal/core/formal/FormalBootstraps.scala

```diff
@@ -118,7 +119,8 @@ case class SpinalFormalConfig(
   }
 
   def withConfig(config: SpinalConfig): this.type = {
-    _spinalConfig = config
+    _spinalConfig = config.includeFormal
```

Comment:
How about only keep this line to help user easily work around the config problem?

Reply:
Multiclock features of formal tools only good for the design which is really have several clocks. It might work slower.

Reply:
You have a actually working alternative with ASYNC reset without multiclock?

Reply:
> You have a actually working alternative with ASYNC reset without multiclock?

The tester in located in tester/ subproject are all use ASYNC reset as default. The SYNC reset has not been tested.
However, although the design itself use ASYNC reset, the verification actually run is still in SYNC manner.
So I think it doesn't matter.